### PR TITLE
New package: glog (c++ logging library)

### PR DESCRIPTION
**Pull request by jprjr**

### CODE REVIEW

File: srcpkgs/glog/template [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Template build file for 'glog'
+pkgname=glog
+version=0.3.4
+revision=1
+build_style=gnu-configure
+configure_args="--prefix=/usr"
```
Comment (Gottox) [inline srcpkgs/glog/template]: Not needed for `gnu-configure`. Remove that line

**Comment by jprjr** [comment]
fixed/rebased/pushed

**Comment by pullmoll** [comment]
There's a warning `=> WARNING: glog-0.3.4_1: libglog.so.0 not found in common/shlibs!`
so you should append
```
libglog.so.0 glog-0.3.4_1
```
to `common/shlibs`.

**Comment by jprjr** [comment]
Added the line to common/shlibs, rebased on master, and pushed

**Comment by pullmoll** [comment]
Sorry, I didn't see this until now: the -doc subpackage should be `noarch=yes`.

**Comment by jprjr** [comment]
Fixed

**Pull request merged**